spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -381,6 +381,9 @@
     end
 
     it "should not contain duplicate tag fields" do
+      # tags contain the current user's tenant's name
+      User.current_user = FactoryGirl.create(:user_with_group)
```

Comment:
This coupling feels weird, and is bothering me.  Why is the `current_user` necessary just to call `model_details`?